examples/tests/sdk_tests/testing/assert.main.w

```diff
@@ -0,0 +1,62 @@
+bring util;
+bring testing;
```

Comment:
it kind of feels like this module needs to be called `assert` and then `assert.equal()`, but then we might need to make sure the compiler still treats `assert()` as a builtin function.

@Chriscbr wdyt?

Reply:
you mean something like "if `blah` refers to a namespace, then `blah(xyz)` desugars into `blah.Util.default(xyz)`"?

Reply:
Yes, one off situation for `assert`.

My ideal name for this module is `assert` but I also want `assert()` to be a built-in function.

Reply:
(personal opinions disclaimer)

> Yes, one off situation for `assert`.

I'm not sure I feel the same about the value of one off exceptions like this.

I think it can be a really healthy design constraint to try and build as much of a language's standard library only using the capabilities that are available in userland. If being unable to have an identifier represent both a namespace and a function is a language capability that you feel is _necessary_ for building Wing-idiomatic libraries and APIs, then IMO it's also a capability we should give to users -- and we should figure out how to design that. (Otherwise, we're making Wing libraries second-class citizens). If the exception doesn't represent an essential capability for building Wing-idiomatic APIs, then it would be preferable to change the name of `assert` (either the built-in function or the SDK module).

> My ideal name for this module is assert but I also want assert() to be a built-in function.

It's not clear to me this is a better DX. Learning programming languages is hard for a lot of people, and saying that `assert` can be both a function and a namespace feels like something that would be surprising to many users, and make Wing harder to learn. (What is the type of `assert` in Wing's type system -- when you mouse over `assert`)? Even though we're spending a lot of time designing Winglang, I think it's important to design the language with empathy for people that can't / don't want to maintain all of the exceptions and edge cases of a language in their head, since this makes it harder to pick up the language and get productive in a new or existing Wing codebase.

I'd compare this a little bit to operator overloading -- this is a capability that many languages add because of the ergonomic benefits (why write `bigInt1.add(bigInt2)` when you can let the user write `bigInt1 + bigInt2`?) -- but it's also _avoided_ in many languages because of the ambiguity it creates for people reading code (it can be hard to determine what function the "+" will call -- see https://journal.stuffwithstuff.com/2010/12/31/rethinking-user-defined-operators/). It also reminds me a bit of the weird oddities of JavaScript, like how classes are also functions `new Foo()` does something different from `Foo()`.

Reply:
Why is `assert` a global function in the first place? both https://www.winglang.io/docs/language-reference#12-utility-functions and https://github.com/winglang/wing/issues/105 aren't providing much context.

Reply:
Okay let's leave this testing.Assert for now.

I hear you folks